### ISSUE TITLE
qt5-host-tools conflicts with qt5

### ISSUE THREAD

**Issue by xtraeme** [void-linux/void-packages#12550]
void-666_1

Found by xbps-0.54:

```
ERROR: qt5-host-tools-5.11.3_7: file `/usr/bin/moc-qt5' already installed by package qt5-5.11.3_7.
[DEBUG] [trans] failed to verify transaction files: File exists
[DEBUG] [pkgdb] released ok.
```

**Comment by xtraeme** [comment]
```
[juan@leysa void-packages]$ xlocate moc-qt5
qt5-5.11.3_7    /usr/bin/moc-qt5 -> /usr/lib/qt5/bin/moc
qt5-5.11.3_7    /usr/bin/moc-qt5 -> /usr/lib/qt5/bin/moc
qt5-host-tools-5.11.3_7 /usr/bin/moc-qt5 -> /usr/lib/qt5/bin/moc
qt5-host-tools-5.11.3_7 /usr/bin/moc-qt5 -> /usr/lib/qt5/bin/moc
[juan@leysa void-packages]$
```

**Comment by Johnnynator** [comment]
Affects:
```
/usr/bin/rcc-qt5 -> /usr/lib/qt5/bin/rcc
/usr/bin/uic-qt5 -> /usr/lib/qt5/bin/uic
/usr/bin/moc-qt5 -> /usr/lib/qt5/bin/moc
/usr/bin/qdbuscpp2xml-qt5 -> /usr/lib/qt5/bin/qdbuscpp2xml
/usr/bin/qdbusxml2cpp-qt5 -> /usr/lib/qt5/bin/qdbusxml2cpp
/usr/bin/qlalr-qt5 -> /usr/lib/qt5/bin/qlalr
```

**Comment by Johnnynator** [comment]
The issue is there https://github.com/void-linux/void-packages/blob/master/srcpkgs/qt5/template#L1058
It checks if usr/bin/*-qt5 is a regular file and therefore ignores symlinks

**Comment by pullmoll** [comment]
... and in https://github.com/void-linux/void-packages/blob/2e01ac8d7952a94f5427ed87c52f27069dac746e/srcpkgs/qt5/template#L1107

**Comment by xtraeme** [comment]
ERROR: qt5-host-tools-5.11.3_7: file `/usr/bin/lconvert-qt5' already installed by package qt5-tools-5.11.3_7.

**Comment by xtraeme** [comment]
Complete list of conflicting files this time:

ERROR: qt5-host-tools-5.11.3_7: file `/usr/bin/lconvert-qt5' already installed by package qt5-tools-5.11.3_7.
ERROR: qt5-host-tools-5.11.3_7: file `/usr/bin/lrelease-qt5' already installed by package qt5-tools-5.11.3_7.
ERROR: qt5-host-tools-5.11.3_7: file `/usr/bin/lupdate-qt5' already installed by package qt5-tools-5.11.3_7.

**Comment by xtraeme** [comment]
Note that xbps-0.54 is really strict with conflicting files and at this moment qt5 packages are broken.

Easy workaround until qt5 is fixed (needs xbps>=0.54):

```
$ echo 'ignorepkg=qt5-host-tools' > /etc/xbps.d/00-ignore-qt5-host-tools.conf
```

**Comment by xtraeme** [comment]
I added `-I, --ignore-file-conflicts` to xbps-install(1) for 0.55.

When ready I'll change xbps-src to use -I to not abort when file conflicts are detected.

This will allow us to catch all conflicting pkgs without interrumping the current workflow.
